src/samtools/flagstat/config.vsh.yaml

```
@@ -0,0 +1,50 @@
+name: samtools_flagstat
+namespace: samtools
+description: Counts the number of alignments in SAM/BAM/CRAM files for each FLAG type.
+keywords: [ stats, mapping, counts, bam, sam, cram ]
+links:
+  homepage: https://www.htslib.org/
+  documentation: https://www.htslib.org/doc/samtools-flagstat.html
+  repository: https://github.com/samtools/samtools.git
```

You can omit the `.git` →

```suggestion
  repository: https://github.com/samtools/samtools
```